app/assets/javascripts/discourse/app/components/quote-button.js

```diff
@@ -378,11 +379,13 @@ export default Component.extend(KeyEnterEscape, {
 
       schedule("afterRender", () => {
         if (this.site.mobileView) {
-          this.element.style.left = `${
-            (window.innerWidth - this.element.clientWidth) / 2
-          }px`;
+          this.textRange = document.querySelector("#main-outlet");
+          this._popper?.update();
         }
-        document.querySelector("#fast-edit-input")?.focus();
+        discourseLater(() => {
```

Comment:
Does this work reliably on iOS? I hesitate to introduce timeouts before focusing on elements, Safari on iOS tends to ignore them. 

Can you try this with `next()` instead, does that also work?

Reply:
Ah yes you're right iOS seems to be ignoring it (same for `next()`, `setTimeout()` or `discourseLater()`)

When we use the `next()`/`discourseLater()`:
It sets the fast edit input position correctly, but it focuses on the text field without bringing up the keyboard. You need to tap once to bring up the keyboard but things are positioned correctly.

Without these, it focuses immediately and brings up the keyboard but the fast edit input's position is broken.

Reply:
Yeah, I don't think there is a way to have our cake and eat it (i.e. to position the card properly and have focus set on the input). 

I think what you have here is good enough, we can't really solve iOS's issue with moving elements when input is focused.